### PR TITLE
hack: Enable TOML++ and CLI11 when system libs used

### PR DESCRIPTION
**Pull request by nbolton**
:exclamation: Not intended to land; no review needed

Should be superseded by #7710

### CODE REVIEW

File: src/test/integtests/deskflow/ConfigTests.cpp [hunk 1]
```diff
@@ -60,6 +60,9 @@ TEST(ConfigTests, load_fileDoesNotExist_returnsFalse) {
   ASSERT_FALSE(result);
 }
 
+// HACK: Fails on FreeBSD:
+// "Error while parsing key-value pair: encountered end-of-file".
+#ifndef __FreeBSD__
```
Comment (nbolton) [inline src/test/integtests/deskflow/ConfigTests.cpp:65]: @sithlord48 Heads up.

File: src/lib/deskflow/CMakeLists.txt [hunk 2]
```diff
@@ -54,6 +54,14 @@ if(UNIX)
...
     server)
 
+  if(CLI11_FOUND)
+    target_link_libraries(${lib_name} CLI11::CLI11)
+  endif()
+
+  if(tomlplusplus_FOUND)
+    target_link_libraries(${lib_name} tomlplusplus::tomlplusplus)
+  endif()
+
```
Comment (nbolton) [inline src/lib/deskflow/CMakeLists.txt:64]: @sithlord48

File: src/test/integtests/deskflow/ConfigTests.cpp [hunk 3]
```diff
@@ -60,6 +60,9 @@ TEST(ConfigTests, load_fileDoesNotExist_returnsFalse) {
   ASSERT_FALSE(result);
 }
 
+// HACK: Disable on FreeBSD and macOS, because of error:
+// "Error while parsing key-value pair: encountered end-of-file".
+#if !defined(__FreeBSD__) && !defined(__APPLE__)
```
Comment (nbolton) [inline src/test/integtests/deskflow/ConfigTests.cpp:65]: @sithlord48

**Comment by sithlord48** [comment]
do we need this after #7745 and #7744  land ?

**Comment by nbolton** [comment]
Nope, it can go.

**Pull request closed without merging**